controller/src/main/java/cern/jarrace/controller/jvm/AgentRegistrySpawner.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * © Copyright 2016 CERN. This software is distributed under the terms of the Apache License Version 2.0, copied
+ * verbatim in the file “COPYING”. In applying this licence, CERN does not waive the privileges and immunities granted
+ * to it by virtue of its status as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+package cern.jarrace.controller.jvm;
+
+/**
+ * Interface that is responsible for spawning JVMs executing cern.jarrace.agent.AgentRegistry#main
+ *
+ * @author tiagomr
+ */
+public interface AgentRegistrySpawner {
```

Comment:
Probably I missed something here: Why does the registry have to be in a separate jvm? I think we only have one per controller, or?